Vagrantfile

```diff
@@ -13,10 +13,12 @@ apt-get update -qq
 apt-get install -y -qq --no-install-recommends chrony
 EOS
 
```

Comment:
Totally fine with keeping it using 100% KVM, if Virtualbox is basically unusable. I was thinking from the perspective of a developer who may want to give Promenade a spin without overhauling their Virtualbox installation